website/app.js

```diff
@@ -125,4 +148,17 @@ export async function main() {
       }
     }
   });
+
+  c3.generate({
+    bindto: "#travis-compile-time-chart",
+    data: { columns: travisCompileTimeColumns },
+    axis: {
+      x: {
+        type: "category",
+        categories: travisData.map(d => d.pull_request_number)
+      }
+    }
```

Comment:
Is there a way we can display the y-axis in minutes instead of seconds?

Reply:
yes.